integration/test/API/BrigInternal.hs

```diff
@@ -27,6 +28,28 @@ instance Default CreateUser where
         supportedProtocols = Nothing
       }
 
+iAddClient ::
+  (HasCallStack, MakesValue user) =>
+  user ->
+  AddClient ->
+  App Response
+iAddClient user args = do
+  uid <- objId user
+  req <- baseRequest user Brig Unversioned $ "/i/clients/" <> uid
```

Comment:
I'm not entirely sure why we were using `Unversioned` here, either, perhaps @pcapriotti  you can verify?

Reply:
nvm, I guess this is just because internal APIs are unversioned in general, :D